.github/workflows/pr-jobs.yml

```diff
@@ -390,9 +390,9 @@ jobs:
     steps:
     - uses: actions/checkout@v4.1.0
     - uses: "./.github/actions/pre-steps"
-    - run: COMPILE_WITH_UBSAN=1 OPT="-fsanitize-blacklist=.circleci/ubsan_suppression_list.txt" CC=clang-10 CXX=clang++-10 ROCKSDB_DISABLE_ALIGNED_NEW=1 USE_CLANG=1 make V=1 -j32 ubsan_check
+    - run: COMPILE_WITH_UBSAN=1 CC=clang-10 CXX=clang++-10 ROCKSDB_DISABLE_ALIGNED_NEW=1 USE_CLANG=1 make V=1 -j32 ubsan_check
```

Comment:
I don't know what's going on. I see clang-10 here but clang-13 in the latest CI job

Reply:
<img width="1396" alt="Screenshot 2024-12-19 at 10 57 32 AM" src="https://github.com/user-attachments/assets/4dbe7512-acb0-404e-8241-bd8076a4231d" />

hmm for the UBSAN test I am seeing clang10. clang13 should only be for the TSAN test (which has now been renamed)